modules/worker-utils/src/lib/worker-farm/worker-body.ts

```diff
@@ -21,68 +28,69 @@ const onMessageWrapperMap = new Map();
  */
 export default class WorkerBody {
   /** Check that we are actually in a worker thread */
-  static inWorkerThread(): boolean {
-    return typeof self !== 'undefined' || Boolean(getParentPort());
+  static async inWorkerThread(): Promise<boolean> {
+    return typeof self !== 'undefined' || Boolean(await getParentPort());
```

Comment:
I guess you found that `self` wasn't defined in Node.js workers? Interesting that this became an issue now and not before.

Reply:
Yes